typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -103,4 +103,7 @@ export enum WarpRouteIds {
   // HYPER routes
   HYPER = 'HYPER/arbitrum-base-bsc-ethereum-optimism',
   stHYPER = 'stHYPER/bsc-ethereum',
+
+  // cttp token bridge
+  USDC_MAINNET_CCTP = 'USDC/mainnet-cctp',
```

Comment:
nit: can we keep the key style consistent with the other ones? e.g. `MainnetCctpUSDC` or something